src/gitflow/GitflowImpl.java

```diff
@@ -378,13 +390,28 @@ public GitCommandResult publishHotfix(@NotNull GitRepository repository,
         return run(h);
     }
 
-    private void setUrl (GitLineHandler h, GitRepository repository){
-        ArrayList<GitRemote> remotes = new ArrayList(repository.getRemotes());
+    @Override
+    public List<String> getBranchList(GitRepository repository) {
```

Comment:
How about using getLocalBranchNames in GitflowBranchUtil where needed instead of adding this here?